### ISSUE TITLE
Dropdown and Multiselect: SVG has no accessible name

### ISSUE THREAD

**Issue by riccardinhocala** [primefaces/primereact#8116]
### Describe the bug

Hello,

I'm using Dropdown and MultiSelect in a project that undergoes strict accessibility checks (IBM Equal Access, WAVE, etc.), and I noticed an issue with svg for drodownIcon (and collapseIcon for Dropdown):

Currently, the <svg> element used for the icon has no accessible name, which results in the following accessibility warning:

```
The SVG element has no accessible name
```

Since the dropdown icon is purely decorative, the best practice according to ARIA guidelines would be to hide it from assistive technologies using the aria-hidden="true" attribute.

Or alternatively, please provide a way to customize the icon’s attributes via props.

Thank you.

### System Information

```Shell
primereact 10.9.6
react 19.0.0
```

### Steps to reproduce the behavior

- Use a <Dropdown> or <MultiSelect> component

- Run an accessibility audit (e.g., IBM EAAC)

- Observe the warning about the SVG icon having no accessible name
